SeleniumTestXUnit/Tests/Features/Item/Delete.feature

```diff
@@ -3,23 +3,25 @@ Feature: Item Deletion
 
     Background:
         Given the user is logged in
-            And the user has an existing project
+        And the user has an existing project
 
+    @Draft
     @Acceptance
     Scenario: Delete a pending item succesfully
         And the user has a pending item in the main items section
         When the user hovers over the item
-            And clicks on the menu list 
-            And clicks on the delete option
-        Then the item should be removed from the section 
-            And it should be added to the Recycle Bin section
+        And clicks on the menu list
+        And clicks on the delete option
```

Comment:
There should be only one step to hover over an item, expanding the menu and selecting an option.